src/table.ts

```diff
@@ -30,13 +30,29 @@ import {Mutation} from './mutation';
 import {Row} from './row';
 import {ChunkTransformer} from './chunktransformer';
 import {CallOptions} from 'google-gax';
+import {Bigtable, RequestCallback, OptionInterface} from '.';
+import {Instance} from './instance';
+import {google} from '../proto/bigtable';
 
 // See protos/google/rpc/code.proto
 // (4=DEADLINE_EXCEEDED, 10=ABORTED, 14=UNAVAILABLE)
 const RETRYABLE_STATUS_CODES = new Set([4, 10, 14]);
 // (1=CANCELLED)
 const IGNORED_STATUS_CODES = new Set([1]);
-
+export interface CreateFamilyOptions extends OptionInterface {
+  rule?: Rule;
+}
+export type CreateFamilyCallback = RequestCallback<
+  Family,
+  google.bigtable.v2.IFamily
+>;
+export type CreateFamilyResponse = [Family, google.bigtable.v2.IFamily];
+export interface Rule {
+  age?: {};
+  versions?: number;
+  intersect?: boolean;
```

Comment:
I do not think this property is used in the family.ts file
I believe this should be `rule`